### ISSUE TITLE
Need latest text and sound version for GFL

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#117]
As mentioned in https://github.com/phetsims/friction/issues/145#issue-378221463 I will be meeting with consultants on Thursday afternoon (1 pm MT) and Friday morning (8 am MT).

It would be great to port the latest version of the descriptions to the sonification wrapper **if there is sound that is ready**. I am not sure there is. Tagging @jbphet to verify.

@mbarlow12, if sounds are not ready, I just need a dev link of the latest described version. 

@mbarlow12, no need to worry about complex alerts and no need to make simple alerts perfect, I just want to get some feedback on how a blind student might interpret the qualitative dynamic descriptions that are currently in the PDOM. I will be asking consultants about the planned complex alerts, too.

If there is not enough time to get a a described & sonified version, please just provide a dev link with a described version.

@mbarlow12 if there is a sonified version and of there is time, please coordinate with @jbphet or @zepumph as needed.

Thanks in advance for whatever is possible for Thursday afternoon.

**Comment by terracoda** [comment]
@jbphet, if the cooling sound is the latest, I can use the link above. It is the cooling sound I am most interested in.

**Comment by terracoda** [comment]
@jbphet, oops, the link above is for Friction. 

Are there any sounds ready for Gravity Force Lab?

**Comment by terracoda** [comment]
@jbphet, I found one for Gravity Force Lab Basics:
https://phet-dev.colorado.edu/html/phet-io-wrapper-sonification/1.0.0-dev.83/phet-io-wrapper-sonification/gravity-force-lab-basics/html/gravity-force-lab-basics-sonification.html

**Comment by terracoda** [comment]
@mbarlow12, I am just realizing now that I should have made the description port request for GFL:Basics, and not GFL. I see from the link above that the sonified version does not even have the latest visuals, so I am not sure a port is even possible.

The description in the sonified version above is really out of date, so I won't bother with it.
 
Could you please provide me with a dev link to latest described versions for Gravity Force Lab and Gravity Force Lab Basics in this issues? 

I'll just focus on description feedback for this sim.

**Comment by terracoda** [comment]
@mbarlow12 I posted on the slack sonification channel that it would be a good idea to update the descriptions for GFL:Basics at their next convenience. 

I just need regular dev links for tomorrow.

**Comment by terracoda** [comment]
@mbarlow12 I found the latest described version for GFL:Basics
[https://bayes.colorado.edu/dev/html/gravity-force-lab-basics/1.0.0-dev.29/phet/gravity-force-lab-basics_all_phet.html?a11y](https://bayes.colorado.edu/dev/html/gravity-force-lab-basics/1.0.0-dev.29/phet/gravity-force-lab-basics_all_phet.html?a11y)

Could you post a dev link for the latest GFL regular in this issue for me today, ideally before 12 noon MT.

**Comment by mbarlow12** [comment]
Sorry for the confusion @terracoda here's the latest from master:

https://phet-dev.colorado.edu/html/gravity-force-lab/2.2.0-dev.9/phet/gravity-force-lab_all_phet.html?ea&accessibility

**Comment by jbphet** [comment]
The latest sound design for GFL:Basics is actually the version linked below with "Option 2" selected.  It's enclosed in an iframe, so that might not work for the current needs, but it is the actual latest.

https://phet-dev.colorado.edu/html/phet-io-wrapper-sonification/1.0.0-dev.84/phet-io-wrapper-sonification/gravity-force-lab-basics/html/gravity-force-lab-basics-sonification-options.html?sonificationFile=sonificationOptions&accessibility

**Comment by terracoda** [comment]
Thanks @jbphet and @mbarlow12. @jbphet, sadly I missed your post, and the older version https://github.com/phetsims/gravity-force-lab/issues/117#issuecomment-436721909, but it was not accessible with JAWS. 

I will have to do some thinking as to why. It might be the iframe, but I am not sure. I will send links in advance to make sure they can be accessed before the next sessions.

Again, thanks to both of you, and @zepumph, too.
